paprika/tests/test_restraints.py

```diff
@@ -1089,7 +1102,9 @@ def test_extract_guest_restraints():
     r.initialize()
     restraints.append(r)
 
-    guest_restraints = extract_guest_restraints(structure, restraints, "BUT")
+    guest_restraints = extract_guest_restraints(
+        structure, restraints, "BUT", return_type="list"
```

Comment:
Do you need to test the dict return too?

Reply:
Yes, I think we need to do that. Thanks for pointing this out.